XcodeServerSDK/Server Entities/Repository.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  Repository.swift
+//  XcodeServerSDK
+//
+//  Created by Mateusz Zając on 28.06.2015.
+//  Copyright © 2015 Honza Dvorsky. All rights reserved.
+//
+
+import Foundation
+
+public struct Repository {
```

Comment:
So far we have all the other types as `class`. Is there a particular reason to start using `struct`s instead?